doc: cephfs: improve documentation of "ceph nfs cluster create" and "ceph fs volume create" commands (#36125)
ceph/ceph: doc/cephfs/fs-nfs-exports.rst

.. code:: bash

>     $ ceph nfs cluster create cephfs <clusterid> [<placement>]

Reviewer: We intend to use the same interface for rgw. So don't remove the type.
```suggestion
    $ ceph nfs cluster create <type> <clusterid> [<placement>]
```

Author: done - please take another look

.. code:: bash

    $ ceph nfs cluster create <type> <clusterid> [<placement>]

This creates a common recovery pool for all NFS Ganesha daemons, new user based on

> cluster_id, and a common NFS Ganesha config RADOS object. It can also bring up

Reviewer: It uses the orch module to create daemons
```suggestion
cluster_id, and a common NFS Ganesha config RADOS object. It brings up
```

Ganesha daemons using the enabled ceph-mgr orchestrator module (see

> :doc:`mgr/orchestrator`), e.g. cephadm.

Author: /home/jenkins-build/build/workspace/ceph-pr-docs/doc/cephfs/fs-nfs-exports.rst:22: WARNING: unknown document: mgr/orchestrator

    $ ceph nfs export create cephfs <fsname> <clusterid> <binding> [--readonly] [--path=/path/in/cephfs]

> This creates export rados objects containing the export block.

Reviewer 2: ```suggestion
This creates export RADOS objects containing the export block.
```

    $ ceph nfs export create cephfs <fsname> <clusterid> <binding> [--readonly] [--path=/path/in/cephfs]

This creates export RADOS objects containing the export block.

> <fsname> is the name of the FS volume used by the NFS Ganesha cluster that will

Reviewer: ```suggestion
``fsname`` is the name of the FS volume used by the NFS Ganesha cluster that will
```
